ibis/backends/bigquery/tests/unit/test_compiler.py

```diff
@@ -470,14 +470,15 @@ def test_range_window_function(alltypes, window, snapshot):
     "preceding",
     [
         param(5, id="five"),
-        param(ibis.interval(nanoseconds=1), id="nanos"),
+        param(ibis.interval(nanoseconds=1), id="nanos", marks=pytest.mark.xfail),
```

Comment:
Was this ever correctly passing?

Reply:
It successfully compiled but would fail to execute.